samza-core/src/main/scala/org/apache/samza/metrics/reporter/MetricsSnapshotReporter.scala

```diff
@@ -160,12 +171,9 @@ class MetricsSnapshotReporter(
         }
       }
     }
-
-
     debug("Finished flushing metrics.")
   }
 
-
```

Comment:
Unrelated to PR but useful: Can you change the info log below "Blacklisted metric %s because it matched ..." to either only be logged once with all metrics or logged at debug?

Reply:
Makes sense. Fixed the log level.